src/css/components/Nav.scss

```diff
@@ -0,0 +1,71 @@
+
+.Nav {
+  background-color: color-neutral(1);
+  display: flex;
+  flex-direction: column;
+
+  a, .nav-item {
+    color: color-neutral(9);
+  }
+
+  .Icon {
+    height: 1.2em;
+    width: 1.2em;
+  }
+
+  &.is-collapsed {
+    max-width: 64px;
+    .nav-item-title {
+      display: none !important;
+    }
+
+    .nav-header {
+      padding: 16px;
+    }
+
+    .nav-collapse {
+      .Icon {
+        transform: rotate(180deg);
+      }
+    }
+  }
+
+  .nav-header {
```

Comment:
Ideally we'd unnest everything from .Nav that doesn't need to be descendent or attached (like the `is-collapsed` styles and pseudo elements). The CSS will render (ever so infinitessimally) faster if it just has the one element to parse. Also Can these inherit the Title case `Nav` pretty please, eg `Nav-header` 🙏